changes/ticket32614

```diff
@@ -0,0 +1,6 @@
+  o Major bugfixes (onion service):
+    - If an rendevous point fails multiple times, we mark the intro point
```

Comment:
Changes file should mention that the fail should be "time-out" related.
Also, "Previously, we kept rendezvous points even if they failed open", do you mean to say "intro points"?